src/libcore/num/mod.rs

```diff
@@ -993,6 +1046,34 @@ $EndFeature, "
             }
         }
 
+        doc_comment! {
+            concat!("Wrapping Euclidean division. Computes `self.div_euc(rhs)`,
+wrapping around at the boundary of the type.
+
+The only case where such wrapping can occur is when one divides `MIN / -1` on a signed type (where
```

Comment:
The wording here feels a little wonky. How about:

_Wrapping will occur in `MIN / -1` on a signed type (where `MIN` is the negative minimal value for the type). This is equivalent to `-MIN`, a positive value that is too large to represent in the type. In this case, this method returns `MIN` itself._

Reply:
I almost copy-pasted that documentation from `wrapping_div`. Should I update the documentation there as well or should that be a separate PR?